assets/dev/js/editor/editor-base.js

```diff
@@ -506,7 +514,16 @@ export default class EditorBase extends Marionette.Application {
 	initPreviewView( document ) {
 		elementor.trigger( 'document:before:preview', document );
 
-		const preview = new Preview( { el: document.$element[ 0 ], model: elementor.elementsModel } );
+		this.previewView = this.renderPreviewFromJSONModel( document.$element[ 0 ],	elementor.elementsModel );
```

Comment:
calling it "render" seems kinda weird when it also returns a value.
Not sure if I have a better name though... WDYT?

Reply:
Done